src/backend/router/index.ts

```diff
@@ -3,17 +3,33 @@ import * as trpcNext from "@trpc/server/adapters/next";
 import { z } from "zod";
 import { prisma } from "@/backend/utils/prisma";
 
-export const appRouter = trpc.router().query("get-random-card", {
-  input: z
-    .object({
-      text: z.string().nullish(),
-    })
-    .nullish(),
-  async resolve({ input }) {
-    const cardFromDb = await prisma.card.findFirst();
-    return { front: cardFromDb?.front };
-  },
-});
+export const appRouter = trpc
+  .router()
+  .query("get-decks", {
+    input: z
+      .object({
+        text: z.string().nullish(),
+      })
+      .nullish(),
+    async resolve({ input }) {
```

Comment:
should we just get rid of this input for now?

Reply:
Tested locally. Works fine without any input! Down to delete for now.

Reply:
Fixed in 589075a2bb32f1f0d01892b214e0a8262e1a3ff6